tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -453,11 +459,38 @@ export class WebGPUBackend extends KernelBackend {
 
     const encoder = this.device.createCommandEncoder();
     const pass = encoder.beginComputePass();
+    if (shouldTimeProgram) {
+      if (env().getBool('WEBGPU_QUERY_API_ENABLED')) {
+        querySet = this.device.createQuerySet({
```

Comment:
Can it be put out of `compileAndRun` and just create once?

Reply:
Yes, Done